components/alert/alert.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="shortcut icon" href="../../assets/Wings.svg" />
-    <link rel="stylesheet" href="./alert.css" />
+    <link rel="stylesheet" href="../../utilities/doc.css" />
     <title>Docs | Alert</title>
```

Comment:
Instead of docs in the title you can use your library name

Reply:
yeah i will change it. Thanks!